src/main/java/uk/nhs/hee/trainee/details/api/TraineeProfileResource.java

```diff
@@ -147,15 +148,13 @@ public ResponseEntity<String> getVerifiedTraineeIds(@NotNull @RequestParam Strin
   }
 
   /**
-   * Get the trainee email for a trainee TIS ID.
+   * Get the account details for a trainee TIS ID.
    *
    * @param tisId The TIS ID to search by.
-   * @return The matching trainee email.
+   * @return The matching trainee user account details, or not found if not found.
    */
-  @GetMapping("/trainee-email/{tisId}")
```

Comment:
I have set this is a bugfix update, rather than a breaking change, since the original PR which created this API never made it to production (https://github.com/Health-Education-England/tis-trainee-details/pull/395)